spec/models/shopify_import/client_spec.rb

```diff
@@ -1,57 +1,92 @@
 require 'spec_helper'
 
 RSpec.describe ShopifyImport::Client, type: :model do
-  describe '.instance' do
-    context 'with valid credentials' do
-      let(:credentials) { { api_key: 'foo', password: 'baz', shop_domain: 'test_shop.myshopify.com' } }
-      let(:client) { described_class.instance }
-      let(:site) { 'https://foo:baz@test_shop.myshopify.com/admin' }
+  describe '#get_connection' do
+    let(:client) { described_class.instance }
 
-      before { client.get_connection(credentials) }
+    after { ShopifyAPI::Base.clear_session }
 
-      it 'creates connection to shopify api' do
-        expect(ShopifyAPI::Base.site.to_s).to eq site
+    context 'with api_key and password' do
+      let(:credentials) do
+        {
+          api_key: '0a9445b7b067719a0af024610364ee34', password: '800f97d6ea1a768048851cdd99a9101a',
```

Comment:
@Argonus What are the downsides? The only difference between this and dummy data is that:

1) When this works, it provides much easier to use interface - you can actually record cassettes, no need to "fake record" on some other shop and then manually edit the yml file. This it the real data. Can't get any more real than this.

2) You can re-record the cassettes.

3) You can create data that's really good for testing (Product with no variants, Product with single variant, Smart Collections, Orders in various states, etc.)

If Shopify would provide a test API, we would be using that. Since they don't, this is the closest we can get.